app/src/main/java/org/schabi/newpipe/player/helper/OldMediaCodecSelector.java

```diff
@@ -0,0 +1,33 @@
+package org.schabi.newpipe.player.helper;
+
+import androidx.annotation.NonNull;
+
+import com.google.android.exoplayer2.mediacodec.MediaCodecInfo;
+import com.google.android.exoplayer2.mediacodec.MediaCodecSelector;
+import com.google.android.exoplayer2.mediacodec.MediaCodecUtil;
+
+import java.util.Collections;
+import java.util.List;
+
+public final class OldMediaCodecSelector implements MediaCodecSelector {
```

Comment:
Maybe add a javadoc for this class, possibly with references to an exoplayer issue

Reply:
Not applicable anymore, due to our decision to remove this setting entirely (see https://github.com/TeamNewPipe/NewPipe/issues/7599#issuecomment-1453663276).